test/extensions/filters/listener/proxy_protocol/proxy_protocol_test.cc

```diff
@@ -1035,6 +1073,58 @@ TEST_P(ProxyProtocolTest, PartialRead) {
   disconnect();
 }
 
+TEST_P(ProxyProtocolTest, PartialV1ReadWithAllowNoProxyProtocol) {
+
```

Comment:
nit, useless space line

Reply:
addressed in https://github.com/envoyproxy/envoy/pull/18951/commits/166746107b979696e7c44b914c897db19e01c442